src/common/meta/src/wal/options_allocator.rs

```diff
@@ -79,13 +104,30 @@ pub fn build_region_wal_options(
     Ok(regions.into_iter().zip(wal_options).collect())
 }
 
-/// Builds a wal options allocator.
-// TODO(niebayes): implement.
-pub async fn build_wal_options_allocator(
-    config: &WalConfig,
-    kv_backend: &KvBackendRef,
-) -> Result<WalOptionsAllocator> {
-    let _ = config;
-    let _ = kv_backend;
-    Ok(WalOptionsAllocator::default())
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::kv_backend::memory::MemoryKvBackend;
+
+    // Tests the wal options allocator could successfully allocate raft-engine wal options.
+    // Note: tests for allocator with kafka are integration tests.
+    #[tokio::test]
+    async fn test_allocator_with_raft_engine() {
```

Comment:
The test is not enough, we don't test kafka allocator at all.

Reply:
Will add more integration tests in the following PR.